Update entry-machines.md (#101)
Espresso-Aficionados/website: recommendations/entry-machines.md

#### Pros:
- Very affordable; lowest cost for a new machine to produce great espresso
- PID Controlled temp (single setting, 200F)
- Integrated OPV set to 9 bar
- Comes with single and dual wall baskets

#### Cons:
- Pre-infusion is not as flexible as Breville thermoblock machines (limited to 10 seconds)

### **Breville Bambino Plus** ([$500](https://www.breville.com/us/en/products/espresso/bes500.html))
### This is the same as  Bambino with slightly better build quality and the addition of an automatic milk steaming wand with adjustable temperature and texture.
#### Pros:

> - Same as Bambino including:

Reviewer: should this have a the in it?
```suggestion
- Same as the Bambino including:
```

> - A robust automatic steaming wand, making steaming a foolproof process

Reviewer: Grammarly suggestion

```suggestion
- A robust automatic steaming wand that makes steaming a foolproof process
```

- 64oz water jug compared to 47 oz of the Bambino
- Comes with cleaning tool/disc for easier maintenance

#### Cons:
- Higher cost than the regular bambino

> - Pre-infusion is not as flexible as Breville thermoblock machines (limited to 10 seconds)

Reviewer: isn't this the same as the regular bambino? If so why not mention it that way?

```suggestion
- Pre-infusion is the same as the regular Bambino (limited to 10 seconds)
```

Author: the two bambinos are the same (and the same to other thermojets); this is a con compared to an infuser or BBE purchase

Reviewer: correct, but does it need to be noted twice if it's the same as above for the regular Bambino?

Author: oh I see what you mean, got it
